opensource/doc-style.md

```diff
@@ -221,14 +221,14 @@ When writing CLI examples, give the user hints by making the examples resemble
 exactly what they see in their shell:
 
 * Indent shell examples by 4 spaces so they get rendered as code blocks.
-* Start typed commands with `$ ` (dollar space), so that they are easily
+* Start typed commands with <tt>$ </tt> (dollar space), so that they are easily
   differentiated from program output.
 * Program output has no prefix.
 * Comments begin with # (hash space).
-* In-container shell commands, begin with `$$ ` (dollar dollar space).
+* In-container shell commands, begin with <tt>$$ </tt> (dollar dollar space).
```

Comment:
I cannot take them out. Currently, the text is rendered wrong, as I explained briefly in the commit message.

Go to https://docs.docker.com/opensource/doc-style/, search for "dollar space". There is no white space (only grey) at the left of the opening parenthesis, but there should be.